lib/v2/screens/authentication/import_key/import_key_screen.dart

```diff
@@ -34,62 +36,67 @@ class _ImportKeyScreenState extends State<ImportKeyScreen> {
+                            _keyController.text = clipboardText;
+                            BlocProvider.of<ImportKeyBloc>(context)
+                                .add(OnPrivateKeyChange(privateKeyChanged: clipboardText));
+                            _onSubmitted();
+                          },
                         ),
-                        onPressed: () async {
-                          var clipboardData = await Clipboard.getData('text/plain');
-                          var clipboardText = clipboardData?.text ?? '';
-                          _keyController.text = clipboardText;
-                          _onSubmitted();
+                        onFieldSubmitted: (value) {
+                          BlocProvider.of<ImportKeyBloc>(context).add(OnPrivateKeyChange(privateKeyChanged: value));
+                        },
+                        controller: _keyController,
+                        onChanged: (value) {
+                          BlocProvider.of<ImportKeyBloc>(context).add(OnPrivateKeyChange(privateKeyChanged: value));
                         },
                       ),
-                      controller: _keyController,
-                      onFieldSubmitted: (_) => _onSubmitted(),
-                      validator: (value) {
-                        if (value?.isEmpty == true) {
-                          return 'Private Key cannot be empty'.i18n;
-                        }
-                        return null;
-                      },
-                    ),
-                  ],
+                    ],
+                  ),
                 ),
               ),
-            ),
-            Padding(
-              padding: const EdgeInsets.only(right: 24, left: 24),
-              child: Text(
-                "If you already have a Seeds account-enter active private key and account will be imported automatically.".i18n,
-                style: Theme.of(context).textTheme.subtitle2OpacityEmphasis,
-                textAlign: TextAlign.center,
+              Padding(
+                padding: const EdgeInsets.only(right: 24, left: 24),
+                child: Text(
+                  "If you already have a Seeds account-enter active private key and account will be imported automatically."
+                      .i18n,
+                  style: Theme.of(context).textTheme.subtitle2OpacityEmphasis,
+                  textAlign: TextAlign.center,
+                ),
               ),
-            ),
-            const SizedBox(height: 24),
-            Expanded(child: ImportKeyAccountsWidget()),
-            Padding(
-              padding: const EdgeInsets.all(16),
-              child: FlatButtonLong(title: 'Search'.i18n, onPressed: () => _onSubmitted()),
-            )
-          ],
-        ),
-      ),
+              const SizedBox(height: 24),
+              Expanded(child: ImportKeyAccountsWidget()),
+            ],
+          ),
+        );
+      }),
```

Comment:
I recommend putting a comma here to lower the builder param and look more like a tree everything :)

Reply:
done (: